doc/plugins/url_options.md

```diff
@@ -10,6 +10,8 @@ storages.
 plugin :url_options, store: { expires_in: 24*60*60 }
 ```
 
+If you're using [GCS](https://googleapis.dev/ruby/google-cloud-storage/latest/Google/Cloud/Storage/File.html#signed_url-instance_method) you need `expires:` instead.
```

Comment:
This is a good attempt though I can see this cascading into adding Azure, and whatever other storage services people use which becomes a maintenance nightmare. Also how many other spots in the documentation will this happen.

While it is nice to have the referring links like this, I think the better solution will be to state that `expires_in` is an example specific to AWS S3 and that the reader should refer to their cloud service for appropriate keys.

One example is:

```
plugin :url_options, store: { expires_in: 24*60*60 }  # `expires_in` is a URL option for AWS S3
```

Thoughts?